server/src/main/java/com/lunchtime/models/Users.java

```diff
@@ -0,0 +1,42 @@
+package com.lunchtime.models;
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.List;
+
+
+@Entity
+@Table(name = "users")
+@Data
+public class Users extends BaseEntity {
```

Comment:
You should use singular noun in class name
https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names